python/SNS_ENCRYPTED_TOPIC_CHECK/SNS_ENCRYPTED_TOPIC_CHECK.py

```diff
@@ -0,0 +1,451 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+Rule Name:
+  SNS_ENCRYPTED_TOPIC_CHECK
+
+Description:
+  Checks whether the Amazon Simple Notification Service (SNS) topics are encrypted. The rule is NON_COMPLIANT if SNS topic is not encrypted or encrypted with a different key then "KmsKeyId".
+
+Trigger:
+  Periodic
+
+Reports on:
+  AWS::SNS::Topic
+
+Rule Parameters:
+  KmsKeyId
+    (Optional) Comma-separated list of KMS keys ARN that should used to encrypt the Amazon Simple Notification Service
+
+Scenarios:
+  Scenario: 1
+     Given: No SNS topic is present
+      Then: Return NOT_APPLICABLE
+  Scenario: 2

```

Comment:
Move it to scenario 1

Reply:
Can you please elaborate this change request ? Do you meant, eliminating the first scenario ? Both of these scenarios are different.

Reply:
Switch scenario 1 and 2.